src/components/data_search/DatasetSearchTableConstants.tsx

```diff
@@ -72,22 +73,36 @@ export const makeStudyTableHeaders = (datasets: DatasetTerm[], selected: number[
     piName: '10%',
     dataCustodians: '20%'
   };
-  const datasetIds = datasets.map(dataset => dataset.datasetId);
+  const isSelectable = (dataset: DatasetTerm) => dataset.accessManagement != 'open' && dataset.accessManagement != 'external';
+  const selectableDatasetIds = datasets.filter(isSelectable).map(dataset => dataset.datasetId);
   return [
     {
       label: <Checkbox checked={datasets.length === selected.length}
-        indeterminate={selected.length > 0 && selected.length < datasetIds.length}
-        onClick={() => onSelect(datasetIds.length === selected.length ? [] : datasetIds)}/>,
+        indeterminate={selected.length > 0 && selected.length < selectableDatasetIds.length}
+        onClick={() => onSelect(selectableDatasetIds.length === selected.length ? [] : selectableDatasetIds)}/>,
       sortable: false,
       cellStyle: makeHeaderStyle(studyCellWidths.selected),
       cellDataFn: datasets => {
         const studyDatasetIds = datasets.map(dataset => dataset.datasetId);
         const numberSelected = _.intersection(studyDatasetIds, selected).length;
         const fullySelected = numberSelected === studyDatasetIds.length;
         const indeterminate = numberSelected > 0 && numberSelected < studyDatasetIds.length;
+        const checkboxId = `${datasets[0].study.studyId}-is-selected-checkbox`;
+        const isSelectableStudy = datasets.filter(isSelectable).length === datasets.length;
+        const tooltipText = 'This study contains some open or external datasets. Please select to apply for access on the individual dataset level.';
```

Comment:
The language here threw me for a loop at first. I would suggest something more like ...

> This study contains some open or external datasets. Select the 'View By Datasets' tab to apply for access to datasets in this study.